.gitignore

```diff
@@ -110,6 +110,9 @@ ParameterMetaData.xml
 apm.pdef.xml
 apm.pdef.json
 parameters.edn
+LogMessages.html
```

Comment:
Could we instead move these into the buildlogs directory - or perhaps generate them there in the first place?

Reply:
I think that is good to have on gitignore anyway but I have now move all generated files into buildlogs directory !